src/main/webapp/app/communication/message/message-inline-input/message-inline-input.component.ts

```diff
@@ -20,6 +20,7 @@ import { Course } from 'app/core/course/shared/entities/course.model';
 })
 export class MessageInlineInputComponent extends PostingCreateEditDirective<Post | AnswerPost> implements OnInit {
     private localStorageService = inject(LocalStorageService);
+    private cdr = inject(ChangeDetectorRef);
```

Comment:
I do not think that it is a good practice to abbreviate it
```suggestion
    private changeDectorRef = inject(ChangeDetectorRef);
```

Reply:
Removed in this [commit](https://github.com/ls1intum/Artemis/pull/10799/commits/fd9c5d41e69ea62cdbda1021248d07419f9d1979). This fixed something unrelated and I will just do more cleanly in a seperate PR.